core/trino-spi/src/main/java/io/trino/spi/predicate/SortedRangeSet.java

```diff
@@ -1151,11 +1166,30 @@ public String toString()
 
         public String formatRange(ConnectorSession session)
         {
+            Object lowValue;
+            if (isLowUnbounded()) {
+                lowValue = "min";
+            }
+            else {
+                lowValue = type.getObjectValue(session, lowValueBlock, lowValuePosition);
+            }
+            if (isSingleValue()) {
+                return format("[%s]", lowValue);
+            }
+
+            Object highValue;
+            if (isHighUnbounded()) {
+                highValue = "max";
+            }
+            else {
+                highValue = type.getObjectValue(session, highValueBlock, highValuePosition);
+            }
+
             return format(
-                    "%s%s,%s%s",
+                    "%s%s, %s%s",
```

Comment:
lack of space was intentional here, for case when dealing when a Set of multiple ranges, so that text wrapping, or "visual eye-balling tokenization" can snap ranges easily

Reply:
reverted